tests/jballerina-unit-test/src/test/resources/test-src/functions/functions_with_included_record_parameters_negative.bal

```diff
@@ -279,3 +279,38 @@ function testInvokingFunctionContainingIncludedRecordParamOfTypeOtherThanRecordO
     _ = obj.fn(f = {});
     _ = obj.fn2(f = {});
 }
+
+type Record5 record {
+};
+
+type Record6 record {
+};
+
+function fn5(*Record5 record5) {
+}
+
+function fn6(*Record5 record5, *Record6 record6) {
+}
+
+function testInvokingFunctionWithNamedArgForIncludedRecordParam() {
+    fn5(record5 = {}, j = 2);
+    fn5(j = 2, record5 = {});
+    fn6(record5 = {}, j = 2);
+    fn6(record5 = {}, record6 = {}, j = 2);
+    fn6(j = 2, record5 = {}, record6 = {});
```

Comment:
Do we have tests for scenarios that are OK like
```suggestion
    fn5(j = 2);
    fn6(record5 = {}, record6 = {});
```
?

Reply:
Add a few more tests to cover this.